components/homme/CMakeLists.txt

```diff
@@ -197,7 +197,9 @@ IF (${HOMME_USE_KOKKOS})
   SET (HOMMEXX_EXEC_SPACE "Default" CACHE STRING "Select the kokkos exec space")
```

Comment:
I think we added it so that we could switch backend without reinstalling kokkos (provided that the new backend was built in kokkos). I agree though that it doesn't seem to be very useful, and probably we could do away with it.